problog/test/test_lfi.py

```diff
@@ -67,28 +67,28 @@ def test_func(self, evaluatable="ddnnf"):
 
         if not os.path.exists(examples):
             raise Exception("Evidence file is missing: {}".format(examples))
-        if useparents:
-            try:
-                d = {
-                    "max_iter": 10000,
-                    "min_improv": 1e-10,
-                    "leakprob": None,
-                    "propagate_evidence": True,
-                    "eps": 0.0001,
-                    "normalize": True,
-                    "web": False,
-                    "args": None,
-                }
-                score, weights, names, iterations, lfi = lfi_wrapper(
-                    model, [examples], evaluatable, d
-                )
-                outlines = lfi.get_model()
-            except Exception as err:
-                print(expectedlines)
-                print(err)
-                # This test is specifically for test/lfi/AD/relatedAD_1 and test/lfi/AD/relatedAD_2
-                assert expectedlines == "NonGroundProbabilisticClause"
-                return
+
+        try:
+            d = {
+                "max_iter": 10000,
+                "min_improv": 1e-10,
+                "leakprob": None,
+                "propagate_evidence": True,
+                "eps": 0.0001,
+                "normalize": True,
+                "web": False,
+                "args": None,
+            }
+            score, weights, names, iterations, lfi = lfi_wrapper(
+                model, [examples], evaluatable, d
+            )
+            outlines = lfi.get_model()
+        except Exception as err:
+            print(expectedlines)
+            print(err)
```

Comment:
Can we remove both print lines so that when the exception is expected, it does not get printed to the user?

Reply:
ok..it was only for testing..but I'll remove them